out/opensearchpy/plugins/alerting.pyi

```diff
@@ -0,0 +1,16 @@
+from ..client.utils import NamespacedClient as NamespacedClient, query_params as query_params
```

Comment:
License header for some files is missing, can you please add that?

Reply:
@arnavdas88 The PR cannot be merged unfortunately without the license headers in place. Can you add them in here?